src/Casdoor.Client/Models/CasdoorAccount.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright 2022 The Casdoor Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Text.Json.Serialization;
+
+namespace Casdoor.Client.Models
+{
+    public class CasdoorAccount
```

Comment:
This class may be duplicated, and the name is not accurate.

Reply:
@AsakusaRinne refer to how Go and Java SDKs name it

Reply:
> @AsakusaRinne refer to how Go and Java SDKs name it

In Go SDK it seems there hasn't been a method to reset password yet. In JAVA SDK it's named ```CasdoorAccountService```.

@sagilio Do you prefer to merge it into an existing class such as ```CasdoorUser```?

Reply:
@AsakusaRinne we can name it `CasdoorSdk`

Reply:
@hsluoyz @AsakusaRinne  Other SDKs do not have this model and it is not used in code, we can just delete it.

Reply:
The class has been removed